openbb_terminal/reports/equity.ipynb

```diff
@@ -33,7 +33,17 @@
     "\n",
     "%matplotlib inline\n",
     "matplotlib_inline.backend_inline.set_matplotlib_formats(\"svg\")\n",
-    "warnings.filterwarnings(\"ignore\")"
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "# Detect if prediction capabilities are present. If they are not, disable prediction in the rest of the script\n",
+    "# so that the report can still be generated without prediction results.\n",
+    "predictions = True\n",
+    "try:\n",
+    "    openbb.stocks.pred.models\n",
+    "except Exception as e:\n",
+    "    predictions = False\n",
+    "    from openbb_terminal.rich_config import console, MenuText\n",
+    "    console.print(\"Failed to import predictions package. Please ensure the prediction package is installed and the feature flag is enabled.\", \"\\n\")"
```

Comment:
To do a quick test, I uninstalled tensor flow and re-ran the equity report.  This line didn't actually trigger (namely I don't see this text in the terminal), but the rest ran without throwing any errors.

<img width="1041" alt="Screen Shot 2022-06-02 at 5 02 56 PM" src="https://user-images.githubusercontent.com/18151143/172024617-a2f53608-2b55-4a8a-bb0d-45fc8023917b.png">
s

Reply:
@jmaslek when this notebook runs, it looks like all output is hidden behind a progress/loading bar. I believe it should show up fine if the notebook is done directly though. That's probably not the main use case though, so let's strike these lines for now.
```suggestion
```